go/v1beta1/storage/esutil/client.go

```diff
@@ -579,3 +580,7 @@ func withRefreshBool(o string) bool {
 	}
 	return true
 }
+
+func escapeDocumentId(id string) string {
+	return url.PathEscape(id)
```

Comment:
Originally `Get` was using `url.QueryEscape` for percent-encoding. 

However, the id ends up in the URL's path so `url.PathEscape` seemed like a better fit; additionally spaces are encoded as `+` signs with `url.QueryEscape`, which isn't decoded by Elasticsearch, changing the document id,